### ISSUE TITLE
Minimal Window width is 1044, more than 1024

### ISSUE THREAD

**Issue by InfoLibre** [linuxmint/cinnamon-spices-applets#4971]
```
 * Applet version 3.3.0
 * Cinnamon version 5.8.4
 * Linux Mint Cinnamon 21.2
 ```

**Notify author of applet**
@mockturtl 

**Issue**
Meteo settings minimal window width is 1044px, more than 1024px. It's to large.

![plus de 800 pixels de large](https://github.com/linuxmint/cinnamon-spices-applets/assets/3955499/75f1c06c-ee6f-42c3-8b4e-394579380864)

**Steps to reproduce**
Open Meteo settings window and reduce its width to minimum.

**Expected behaviour**
Minimal Window size should be ideally 800 or 1024.

**Other information**

**Comment by InfoLibre** [comment]
See https://github.com/linuxmint/cinnamon-spices-applets/pull/4972

**Comment by claudiux** [comment]
Ping @Gr3q

**Comment by rcalixte** [comment]
This is resolved by #4972

**Comment by InfoLibre** [comment]
Thank you very much.
